server/parsec/backend/realm.py

```diff
@@ -259,32 +233,6 @@ async def apiv2v3_realm_get_role_certificates(
 
         return authenticated_cmds.v3.realm_get_role_certificates.RepOk(certificates)
 
-    @api
-    async def apiv2_realm_update_roles(
-        self,
-        client_ctx: AuthenticatedClientContext,
-        req: authenticated_cmds.v2.realm_update_roles.Req,
-    ) -> authenticated_cmds.v2.realm_update_roles.Rep:
-        # `realm_update_roles` command is similar between APIv2 and v4+ from the server
-        # point of view.
-        # (from client point of view, server may return `bad_timestamp` response
-        # with some fields missing)
-        # TODO: proper api req/rep conversion
-        return await self.api_realm_update_roles(client_ctx, req)  # type: ignore[return-value, arg-type]
-
-    @api
-    async def apiv3_realm_update_roles(
```

Comment:
APIv3 should stay !

Reply:
See https://github.com/Scille/parsec-cloud/pull/4845/files#r1276297985